README.md

```diff
@@ -32,67 +32,47 @@ var readStream = repo.archive.readFile('hello.txt', function (err, data) {
 var Dat = require('dat-js')
 
 var dat = new Dat()
-var repo = dat.create()
+var archive = dat.create()
 
 console.log('dat url is:', repo.url)
 
 // You can start reading/writing right away
-var writer = repo.archive.createWriteStream('hello.txt')
+var writer = archive.createWriteStream('hello.txt')
 
 writer.write('world')
-writer.end(function () { replicate(repo.url) })
+writer.end(function () { replicate(archive.url) })
 
 function replicate (url) {
   var clone = new Dat()
   var repo = clone.get(url)
 
-  var readStream = repo.archive.createReadStream('hello.txt')
+  var readStream = archive.createReadStream('hello.txt')
   readStream.on('data', function (data) {
     console.log(data.toString()) // prints 'world'
   })
 }
 ```
 
-#### Replication with a websocket gateway
-
-```js
-var Dat = require('dat-js')
-var concat = require('concat-stream')
-var pump = require('pump')
-
-var dat = new Dat({
-  gateway: 'ws://gateway.mauve.moe:3000'
-})
-var repo = dat.get('dat://SOME_ARCHIVE_URL')
-
-var readStream = repo.archive.createReadStream('hello.txt')
-
-pump(readStream, concat(function (data) {
-  console.log(data)
-}))
-```
-
 #### Persisting a created dat and loading it from storage
 
 ```js
 var Dat = require('dat-js')
-var db = require('random-access-idb')('dats')
 
 var dat = new Dat()
 
-var repo = dat.create({
-  db: db
+var archive = dat.create({
+  persist: true
```

Comment:
Can you add a sentence or two on how this works and what happens if you don't use this? (i.e. link to the repos and say what "persist" means in terms of browser storage.

Reply:
Oh I see this in in API section. May be good to mentio here too since it is a whole section on it =)

Reply:
Added a comment relating to it. Should I have a description instead?